src/promptflow-evals/promptflow/evals/synthetic/_conversation/__init__.py

```diff
@@ -85,9 +85,12 @@ def __init__(
                 if isinstance(conversation_starter_content, dict):
                     self.conversation_starter = conversation_starter_content
                 else:
-                    self.conversation_starter = jinja2.Template(
-                        conversation_starter_content, undefined=jinja2.StrictUndefined
-                    )
+                    try:

```

Comment:
do we need a unit test for covering this case?

Reply:
Just did! Thanks :)